guides/common/attributes-titles.adoc

```diff
@@ -48,6 +49,7 @@ ifdef::satellite[]
 :ManagingConfigurationsPuppetDocTitle: Managing Configurations Using Puppet Integration in {ProjectName}
 :PlanningDocTitle: {Project} Overview, Concepts, and Deployment Considerations
 :QuickstartDocTitle: Quickstart
+:UpgradingDocTitle: Upgrading Connected {ProjectName} to {ProjectVersion}
```

Comment:
Is this duplicate to line `28` ?

Reply:
It isn't a duplicate, it's a modification of the title for the `satellite` build. It's correct like this.

Reply:
Hey @mjivraja so we need two separate titles for upstream and Satellite, so line 52 is only for downstream.

Reply:
I overlooked condition. Ack!